src/default_panels/StyleTracesPanel.js

```diff
@@ -477,21 +488,19 @@ const StyleTracesPanel = (props, {localize: _}) => (
     </PlotlySection>
     <TraceTypeSection
       name={_('Text')}
-      traceTypes={[
-        'scatter',
-        'scattergl',
-        'scatterpolar',
-        'scatterpolargl',
-        'barpolar',
-        'pie',
-        'scatter3d',
-        'scatterternary',
-        'bar',
-        'scattergeo',
-        'scattermapbox',
-        'sunburst',
-        'waterfall',
-      ]}
+      traceTypes={allTraceTypes.filter(
+        t =>
+          ![
+            'histogram2d',
+            'histogram2dcontour',
+            'parcoords',
+            'parcats',
+            'sankey',
+            'table',
```

Comment:
do Violin, Box and Histogram support Text??

Reply:
![image](https://user-images.githubusercontent.com/2717746/55923236-96a27600-5bd2-11e9-92b0-4098b4cefd28.png)

Reply:
whaddaya know. ok then